Dockerfile.server

```diff
@@ -59,10 +52,26 @@ RUN BUILD_SEQUENCIAL=1 yarn install --inline-builds \
     && yarn cache clean \
     && rm -rf .yarn/berry
 
-# Call the command directly, without going via npm:
-# https://github.com/nodejs/docker-node/blob/main/docs/BestPractices.md#cmd
-WORKDIR "/home/$user/packages/server"
-CMD [ "node", "dist/app.js" ]
+### Final image
+FROM node:16-bullseye-slim
+
+ARG user=joplin
+RUN useradd --create-home --shell /bin/bash $user
+
+USER $user
+
+COPY --chown=$user:$user --from=builder /build/packages /home/$user/packages
+COPY --chown=$user:$user --from=builder /tini /usr/local/bin/tini
+
+ENV NODE_ENV=production
+ENV RUNNING_IN_DOCKER=1
+EXPOSE ${APP_PORT}
+
+# Use Tini to start Joplin Server:
+# https://github.com/nodejs/docker-node/blob/main/docs/BestPractices.md#handling-kernel-signals
```

Comment:
Do we actually need Tini? What problem does it solve? It's not clear from the linked post.

Reply:
@laurent22 If you launch Joplin Server with the current Docker image using `docker run [...]`, it will ignore termination signals like `SIGINT` (e.g. it won't exit when you press CTRL+C). You have to open a new SSH session and kill the Docker container manually in order to shut it down. It also won't exit gracefully if you start and stop containers using systemd.

See https://www.baeldung.com/linux/sigint-and-other-termination-signals#sigint for details.